pkg/apply/event/event.go

```diff
@@ -198,11 +204,11 @@ func (age ActionGroupEvent) String() string {
 type ApplyEventOperation int
 
 const (
-	ApplyUnspecified ApplyEventOperation = iota
-	ServersideApplied
-	Created
-	Unchanged
-	Configured
+	ApplyUnspecified  ApplyEventOperation = iota
+	ServersideApplied                     // TODO: rename AppliedServerside
```

Comment:
Most of the other event operations are prefixed by something consistent.

Do we want to change these to use Applied/Apply as prefix?

It would make them easier to auto-complete. We can still print them as something without the prefix, because the generator allows specifying the string value as an inline comment.

Reply:
This should probably be punted to a different PR. It will technically break reverse compatibility by changing an existing event.

Reply:
Punted to https://github.com/kubernetes-sigs/cli-utils/issues/502